libs/@local/hql/diagnostics/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+cargo-features = ["edition2024"]
+
+[package]
+name = "hql-diagnostics"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license = "AGPL-3.0"
+publish = false
+
+[dependencies]
+anstyle = "1.0.8"
+anstyle-yansi = "2.0.1"
```

Comment:
is this required by `ariadne` or can we use `antsi`?

Reply:
`ariadne` requires `yansi`, I have been thinking of replacing `antsi` in favor of `anstyle` anyway, it seems to do a lot things right (although does not document the different standards)